openfl-workspace/workspace/plan/defaults/network.yaml

```diff
@@ -4,6 +4,7 @@ settings:
     agg_port                   : auto
     hash_salt                  : auto
     use_tls                    : True
+    enable_remote_attestation  : False
```

Comment:
I would suggest changing this and moving the `enable_remote_attestation` parameter to the `aggregator` and `collaborator` sections so that it can be configured independently; I expect it will be common that only the aggregator can run inside SGX